types/jest-in-case/jest-in-case-tests.ts

```diff
@@ -25,16 +25,16 @@ afterEach(() => {
 test('array', () => {
     const title = 'add(augend, addend)';
 
-    const tester = jest.fn((opts, cb) => {
-        expect(add(opts.augend, opts.addend)).toBe(opts.total);
-    });
-
     const testCases = [
         { name: '1 + 1 = 2', augend: 1, addend: 1, total: 2 },
         { name: '2 + 1 = 3', augend: 2, addend: 1, total: 3 },
         { name: '3 + 1 = 4', augend: 3, addend: 1, total: 4 },
     ];
 
+    const tester: (testCase: typeof testCases[0], cb?: any) => void = jest.fn((opts, cb) => {
```

Comment:
I think that this is incorrect.

As the mock function has two arguments it will be called with done ( it is not optional ) and as such the original toHaveBeenCalledWith expectations are also incorrect and should be 
`expect(tester).toHaveBeenCalledWith(testCases[0],expect.any(Function));`

The actual test does not have the done argument.  You should probably just remove the cb
https://github.com/atlassian/jest-in-case/blob/78fb05fd0a021fb183d0c78f0d13bc65c5dbfae3/test.js#L26

https://github.com/atlassian/jest-in-case/blob/78fb05fd0a021fb183d0c78f0d13bc65c5dbfae3/index.js#L37
`

      if (tester.length > 1) {
        cb = done => tester(testCase, done);
      } else {
        cb = () => tester(testCase);
      }
`

Reply:
I've removed the cb argument